### ISSUE TITLE
Autorun script on raspberry pi (RPI)

### ISSUE THREAD

**Issue by msergencatal** [alphacep/vosk-api#240]
Hi.  I am working on a project about speech recognition with raspberry pi and vosk api. I want to autorun my python script when I plug in the  RPI. I have done autorun configuration on RPI. Script is running when I plug in the PRI but can't read the "model" file. I have specified the file path on the script shown below. 
"
if not os.path.exists("/home/pi/Vosk_API/model"):
    print ("Please download the model from https://alphacephei.com/vosk/models and unpack as 'model' in the current folder.")
    exit (1)
"
I don't get this  message but RPI sent this error below and couldn't autorun my script.
"
ERROR (VoskAPI:ReadConfigFile():parse-options.cc:462) Cannot open config file: model/mfcc.conf
terminate called after throwing an instance of 'kaldi::KaldiFatalError'
  what():  kaldi::KaldiFatalError
"
Script is running without autorun with cmd window but i want to run when i plug in the RPI. Could you please help me to solve this issue? Thank you for your attention.

**Comment by nshmyrev** [comment]
> Cannot open config file: model/mfcc.conf

In your script specify the full path to the model when you load it, not just when you check the files:

https://github.com/alphacep/vosk-api/blob/d666876be17cc4a6f341bf6b0f74e361a308d0fe/python/example/test_simple.py#L19

     model = new Model("/home/pi/Vosk_API/model")

not simply 

     model = new Model("model")

**Comment by msergencatal** [comment]
Thank you for your help. I specified the full file path. Script run but i
have an another error like below
"
  File "/home/pi/Vosk_API/test7.py", line 41, in <module>
    stream = p.open(format=pyaudio.paInt16, channels=1, rate=16000,
input=True, frames_per_buffer=8000)
  File "/usr/lib/python3/dist-packages/pyaudio.py", line 750, in open
    stream = Stream(self, *args, **kwargs)
  File "/usr/lib/python3/dist-packages/pyaudio.py", line 441, in __init__
    self._stream = pa.open(**arguments)
OSError: [Errno -9997] Invalid sample rate
"
I have no errors like this when run with cmd window. I can record a sound
with my mic and play with my headphones. Do you have any idea about this
error ?

**Comment by nshmyrev** [comment]
Try to set sample rate 44100 in pyaudio and in kaldi recognizer. There are issues with plain recording to 16k.

**Comment by msergencatal** [comment]
I solved the problem by change the autorun seetings. it s runing both 16k and 44100 sample rate. Firstly i have tried this autorun method. https://www.youtube.com/watch?v=meeKv3DY9ps&t=208s I have this error with this method. I noticed there was a problem about default playback and recording device . So i dediced to change autorun method and tried this one. https://www.youtube.com/watch?v=cZa1oCSdbUk&t=307s This autorun method runing good with the script. I have no error i can test my script. Maybe this help someone else who have the same problem. Thank you for your attention Nickolay.
